### PR TITLE
[dbart] push picture

### PR DESCRIPTION
**Pull request by sshleifer**

### CODE REVIEW

**Comment by sshleifer** [comment]
This looks obnoxiously large in the diff viewer. Should we do some fancy markdown instead of
```markdown
![DBART](https://github.com/sshleifer/transformers_fork/raw/add-distilbart-pic/examples/seq2seq/distilbart_w_logos.png)
```

**Comment by julien-c** [comment]
The README view is fixed-width on GitHub so you should be fine.

However, I would try to find a more permanent host/URL for your image, I suspect your fork's branch will get deleted at some point.

File: examples/seq2seq/README.md [hunk 1]
```diff
@@ -128,6 +128,7 @@ Here is an example command, but you can do whatever you want. Hopefully this wil
 You can see your wandb logs [here](https://app.wandb.ai/sshleifer/hf_xsum?workspace=user-)
 
 ### DistilBART
+![DBART](https://github.com/sshleifer/transformers_fork/raw/add-distilbart-pic/examples/seq2seq/distilbart_w_logos.png)
```
Comment (JetRunner) [inline examples/seq2seq/README.md:131]: I’ll make you a version that is suitable for README (like the one I used for PABEE). Also I think uploading the image to our own server is a good idea, like this: https://huggingface.co/front/thumbnails/microsoft.png

**Review by JetRunner — CHANGES_REQUESTED** [review]

File: examples/seq2seq/README.md (hunk 1 above)
Comment (JetRunner) [reply examples/seq2seq/README.md:131]: I've sent @sshleifer on Slack

**Comment by JetRunner** [comment]
#5394

**Pull request closed without merging**